src/native-common/PopupContainerView.tsx

```diff
@@ -113,9 +111,13 @@ export class PopupContainerView extends React.Component<PopupContainerViewProps,
     }
 
     render() {
-        var popupView = this.props.activePopupOptions.renderPopup(
-            this.state.anchorPosition, this.state.anchorOffset,
-            this.state.constrainedPopupWidth, this.state.constrainedPopupHeight);
+        const popupView = (this.props.hidden ?
+            this.props.popupOptions.renderPopup('top', 0, 0, 0) :
```

Comment:
We could remember the arguments from when the popup was active and keep passing them when it's not. It doesn't seem to make much difference and probably not worth the added complexity.